### PR TITLE
Education / training history

### LINKED ISSUE #2933: EducationHistoryPage component
https://app.moqups.com/greg@adhocteam.us/l32hJ36b/view/page/a1d771bf5

### PR DESCRIPTION
**Pull request by alexose**
Fixes #2933.

Nothing too fancy here-- There's just a bit of logic involving whether or not to show the GrowableTable form.

### CODE REVIEW

File: src/js/edu-benefits/components/EducationPeriod.jsx [hunk 1]
```diff
@@ -0,0 +1,84 @@
...
+            label="City"
+            name="city"
+            field={period.city}
+            onValueChange={(update) => {onValueChange('city', update);}}/>
+        <ErrorableSelect
+            label="State"
+            name="state"
+            field={period.state}
+            value={period.state}
+            options={states.USA}
```
Comment (jbalboni) [inline src/js/edu-benefits/components/EducationPeriod.jsx]: Do we know if these are US colleges/trainings only?

File: src/js/edu-benefits/components/EducationHistoryFields.jsx [hunk 2]
```diff
@@ -0,0 +1,64 @@
+import React from 'react';
+
+import DateInput from '../../common/components/form-elements/DateInput';
+import GrowableTable from '../../common/components/form-elements/GrowableTable';
+
+import EducationPeriod from './EducationPeriod';
+import { createEducationPeriod } from '../utils/veteran';
+
+import { isValidSection, isValidDate } from '../utils/validations';
```
Comment (jbalboni) [inline src/js/edu-benefits/components/EducationHistoryFields.jsx]: This should be isValidPage

**Comment by jbalboni** [comment]
@alexose Couple minor comments, but LGTM!

File: src/js/edu-benefits/components/EducationPeriod.jsx (hunk 1 above)
Comment (alexose) [inline src/js/edu-benefits/components/EducationPeriod.jsx]: Hrm, I don't know.  I'll follow up with design

**Pull request merged**